spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/core/SendingOperation.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.spring.integration.core;
+
+import java.util.concurrent.CompletableFuture;
+
+
+/**
+ * Operations for sending message to a destination.
+ *
+ * @param <D> the type of message
+ * @author Warren Zhu
+ */
+public interface SendingOperation<D> {
```

Comment:
Why naming the interface name with `-ing` format?

Reply:
Spring also has `SendingOperation`. In the future, we might need to user spring's rather than provide our own. So keep it consistent here.